### DISCUSSION TITLE
execution of a task in sequence

### DISCUSSION THREAD

**Discussion by dariocap** [lllyasviel/Fooocus#2090] [Q&A] [answered]
Can you tell me if it is possible to change something so that at the end of generating my prompt, it automatically reads the second prompt and at the end, it reads the third prompt?
The idea is to automate the creation of a list of prompts without waiting for Fooocus to finish its work, to write the next prompt.

**Comment by Moodpedro** [comment]
I was also looking for the possibility of batch execution of differents prompts and settings... so let's say, I want to make a series of image generating (with totally different tasks), I could leave and the job is done during several hours while doing other things...

In adobe premiere it is solved with an exterior Media Encoder, so from the main program I can create as many tasks as I want, and then the encoder does its job in sequence.

**Comment by mashb1t** [comment, marked as answer]
There is https://github.com/lllyasviel/Fooocus/pull/1773 which you might want to have a look at

**Comment by Moodpedro** [comment]
I really appretiate! Thanks!
